src/lex.cpp

```diff
@@ -450,13 +450,45 @@ void lexer::parse_current_token() {
       if (this->input_[2] == '=') {
         this->last_token_.type = token_type::star_star_equal;
         this->input_ += 3;
+      } else if (this->input_[2] == '/') {
+        // **/ instead of star_star slash, interpret as star, star, slash
```

Comment:
Must fix: `3 **/ banana/` is valid JavaScript. `*` `*` `/` is an incorrect interpretation.